README.md

```diff
@@ -21,7 +21,7 @@ BandsContainer is connected to __Redux__ and has mapped `name => dispatch({ type
 for a single band.
 
 2. Change the structure of the state such that each band has its own id. You
-will also need to pass through the band object(this should include the _id_ and
+will also need to pass through the band object (this should include the _id_ and
```

Comment:
This is great, but the change should be made in a PR to update `master` so that we can rebase solution on `master`. If this is vital, can you open a new PR to do this one in that?